homeassistant/components/soundtouch/manifest.json

```diff
@@ -6,5 +6,6 @@
   "after_dependencies": ["zeroconf"],
   "codeowners": [],
   "iot_class": "local_polling",
-  "loggers": ["libsoundtouch"]
+  "loggers": ["libsoundtouch"],
+  "version": "1.0.0"
```

Comment:
As far as I understand, the version prop is only for custom components, but since this is included in core, no version is needed.

Reply:
Correct, I will remove it. Thank you.
Refernece: https://developers.home-assistant.io/docs/creating_integration_manifest/#version